docs/content/topics/developers.md

```diff
@@ -61,12 +63,16 @@ $ git clone https://github.com/brigadecore/brigade $GOPATH/src/github.com/brigad
 $ cd $GOPATH/src/github.com/brigadecore/brigade
 ```
 
-**Note**: this leaves you at the tip of **master** in the repository where active development
-is happening. You might prefer to checkout the most recent stable tag:
+**Note**: this leaves you at the tip of **master** in the repository where
+active development is happening. You might prefer to checkout the most recent
+stable tag:
 
-- `$ git checkout v1.2.1`
+```console
+$ git checkout v1.2.1
```

Comment:
Especially given that these are v2 docs, should we perhaps be directing would-be contributors to checkout the `v2` branch instead?